src/rtl_tcp.c

```diff
@@ -524,6 +524,7 @@ int main(int argc, char **argv)
 	pthread_cond_init(&cond, NULL);
 	pthread_cond_init(&exit_cond, NULL);
 
+	memset(&hints, 0, sizeof(hints));
```

Comment:
I noted the need for this important fix myself. On Windows, I got the error:
```
local address 127.0.0.1 ERROR - En ugjenopprettelig feil oppstod under et databaseoppslag. .
```

Which is `WSANO_RECOVERY (11003)`. 

PS. note the 2 `.` above since `gai_strerror()` returns one too.

Reply:
oh lovely! I'm glad this also fixes issues on windows!